js/skin.js

```diff
@@ -394,6 +395,7 @@ var Skin = createReactClass({
               seeking={this.state.seeking}
               upNextInfo={this.state.upNextInfo}
               isLiveStream={this.state.isLiveStream}
+              controlBarAutoHide={this.props.skinConfig.controlBar.autoHide}
```

Comment:
Not super important, but I think it might be better to read this directly from skin config rather than have a separate prop. I've struggled with some unit tests that use props that come from two different sources